hexa/workspaces/views.py

```diff
@@ -96,13 +98,37 @@ def credentials(request: HttpRequest, workspace_slug: str = None) -> HttpRespons
     )
 
     # Bucket credentials
-    token, _ = get_short_lived_downscoped_access_token(workspace.bucket_name)
-    env.update(
-        {
-            "WORKSPACE_BUCKET_NAME": workspace.bucket_name,
-            "GCS_TOKEN": token,
-        }
+    token, _expires_in, engine = get_short_lived_downscoped_access_token(
+        workspace.bucket_name
     )
+    if engine == "s3":
```

Comment:
This could be moved in a `get_environment_variables` method or something similar in the engine classes